firefox_accounts/firefox_accounts.model.lkml

```diff
@@ -3,8 +3,10 @@ label: "Firefox Accounts"
 include: "//looker-hub/firefox_accounts/views/fxa_first_seen_table.view.lkml"
 include: "//looker-hub/firefox_accounts/views/login_funnels.view.lkml"
 include: "//looker-hub/firefox_accounts/explores/*"
-include: "views/*.view.lkml"
-include: "explores/*.explore.lkml"
+include: "views/events.view.lkml"
+include: "views/growth_accounting.view.lkml"
```

Comment:
I believe this should still include `views/all_events.view.lkml` and `views/auth_events.view.lkml`, because otherwise with this change those view refinements are no longer being applied.
```suggestion
include: "views/all_events.view.lkml"
include: "views/auth_events.view.lkml"
include: "views/events.view.lkml"
include: "views/growth_accounting.view.lkml"
```